homeassistant/components/device_tracker/owntracks.py

```diff
@@ -22,20 +23,28 @@
 
 LOCATION_TOPIC = 'owntracks/+/+'
 EVENT_TOPIC = 'owntracks/+/+/event'
+WAYPOINT_TOPIC = 'owntracks/{}/{}/waypoint'
```

Comment:
The second `{}` is used to set the device in test_owntracks, so kept the format generic here.